packages/context/src/context-view.ts

```diff
@@ -161,10 +161,10 @@ export class ContextView<T = unknown> extends EventEmitter
  */
 export function createViewGetter<T = unknown>(
   ctx: Context,
-  bindingFilter: BindingFilter<T>,
+  bindingFilter: BindingFilter,
   session?: ResolutionSession,
 ): Getter<T[]> {
-  const view = new ContextView(ctx, bindingFilter);
+  const view = new ContextView<T>(ctx, bindingFilter);
```

Comment:
Similarly here. Before this change, it was possible to infer `T` of `ContextView` from `bindingFilter` type. After this change, `new ContextView(...)` always treats `T` as `unknown`.